packages/website/src/components/RulesTable/index.tsx

```diff
@@ -231,7 +231,7 @@ export default function RulesTable(): React.JSX.Element {
             setMode={(newMode): void =>
               changeFilter('typeInformation', newMode)
             }
-            label={`${TYPE_INFORMATION_EMOJI} requires type information`}
+            label={`${TYPE_INFORMATION_EMOJI} type checked`}
```

Comment:
@Josh-Cena from https://github.com/typescript-eslint/typescript-eslint/pull/7993#pullrequestreview-1749230642 and https://github.com/typescript-eslint/typescript-eslint/pull/7993#pullrequestreview-1749230642:

> Is the notion of "type checked" mentioned anywhere? To me it could be interpreted as "the rule itself is type checked" and that other rules are like ESLint core rules (just JavaScript running rogue). Me playing devil's advocate though.

> Mmmm, I see that the preset is called recommended-type-checked. Maybe that counts as one occasion.

Yeah I think the phrase "type checked" is a good one for us to repeat. It's pretty succinct and I can't think of anything more clear.